packages/core/src/mocks.ts

```diff
@@ -21,7 +21,7 @@ export class MockEIP1193Provider extends EventEmitter {
       case 'eth_requestAccounts':
         return Promise.resolve(this.eth_requestAccounts(this.accounts))
       default:
-        throw new Error()
+        throw new Error(`Method not supported: ${JSON.stringify(x)}`)
```

Comment:
```suggestion
        throw new Error(`Method not supported on mock: ${JSON.stringify(x)}`)
```

Reply:
Just improve this error message so it's obvious that the reason it's failing is because it is not mocked. Otherwise users may have a harder time debugging failing tests.